### PR TITLE
Provide option to allow the response text when using abort handler

### PR DESCRIPTION
**Pull request by JeffBeltran**
Hello, just wanted the ability to pass a message string for middleware error handling. If value is null it will still work as before but this way it allows us to customize the abort() call.

### CODE REVIEW

**Comment by santigarcor** [comment]
Would be nice to handle the message in case it's not set in the configuration, then I don't have to make a 5.3 release and just a minor release with this in it.

**Comment by JeffBeltran** [comment]
ah shoot yeah i missed that, ok i'll get a change pushed up so we don't introduce any breaking changes.

File: src/Middleware/LaratrustMiddleware.php [hunk 1]
```diff
@@ -46,7 +46,7 @@ protected function unauthorized()
         $handler = Config::get("laratrust.middleware.handlers.{$handling}");
 
         if ($handling == 'abort') {
-            return App::abort($handler['code']);
+            return App::abort($handler['code'], $handler['message']);
```
Comment (santigarcor) [inline src/Middleware/LaratrustMiddleware.php]: it could be something like `return App::abort($handler['code'], $handler['message'] ?? '');`

Comment (JeffBeltran) [reply src/Middleware/LaratrustMiddleware.php]: yup, clean enough for me pushing this up now

**Comment by JeffBeltran** [comment]
@santigarcor pushed up recommended change. Thanks!

**Pull request merged**